js/project-list.js

```diff
@@ -0,0 +1,47 @@
+/* Project List */
```

Comment:
Please fix all "Project List" related notes and variable names to "Project List Item".  Thanks!